packages/common/src/store/accounts.slice.ts

```diff
@@ -120,11 +124,12 @@ export const {
   fetchAddresses,
   fetchFailed,
   fetchReset,
-  setGeneratedAccount
+  setGeneratedAccount,
+  setGeneratedAccountPersistent
 } = slice.actions;
 
 export const addSavedAccounts = createAction<boolean>(`${sliceName}/addSavedAccounts`);
-export const addGeneratedAccount = createAction<boolean>(`${sliceName}/addGeneratedAccount`);
+export const addGeneratedAccount = createAction(`${sliceName}/addGeneratedAccount`);
```

Comment:
Can you specify a type

Reply:
What do you mean? There is no payload for this action anymore